packages/node_modules/nav-frontend-skjema/src/bekreft-checkboks-panel.tsx

```diff
@@ -84,7 +91,8 @@ export class BekreftCheckboksPanel extends React.Component<BekreftCheckboksPanel
     className: PT.string,
     checked: PT.bool.isRequired,
     label: PT.string.isRequired,
-    onChange: PT.func.isRequired
+    onChange: PT.func.isRequired,
+    feil: PT.bool
```

Comment:
`skjemaelementFeilmeldingShape` skal det vel være her, istedetfor `PT.bool`

Reply:
Takk for tips. Den så jeg ikke.